packages/core/src/Agentica.ts

```diff
@@ -216,7 +216,9 @@ export class Agentica {
       prompt: props.prompt,
 
       // HANDLERS
-      dispatch,
+      dispatch: async (event: IAgenticaEvent) => {
```

Comment:
Rollback this one and change the context type please.

Reply:
sure, but already relaesed it(0.8.2)

i'll go home and modify it

Reply:
Did it